timemachine/potentials/summed.py

```diff
@@ -33,7 +33,7 @@ def summed_potential(
         shapes of the parameter array input for each potential term (must be same length as U_fns)
     """
     assert len(U_fns) == len(shapes)
-    sizes = np.prod(shapes, axis=1)
+    sizes = [np.prod(shape) for shape in shapes]
     # np.split expects indices, must increment sizes to be indices
     split_indices = np.cumsum(sizes)
```

Comment:
Question: would it be redundant to `assert params.shape == (sum(sizes),)`?

(May not add anything, since a stronger check is already implicit in np.split(...).reshape(...) below.)

Reply:
I think this is a good suggestion. As you say it fails if the condition is not met, but the issue is immediately obvious. Added assertions on input parameters in https://github.com/proteneer/timemachine/pull/1075/commits/d9dc6d2fec7c17d4605f58cc2d6c6d0529ad878c